Week_03/home.html

```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="css/styles.css">
     <link rel="stylesheet" href="normalize.css">
-    <title>Home</title>
+    <title>App Home</title>
```

Comment:
Hey actually we should leave this alone
```suggestion
    <title>Home</title>
```